core/src/main/java/io/questdb/cutlass/http/DefaultRejectProcessorFactory.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cutlass.http;
+
+import io.questdb.cutlass.http.processors.RejectProcessor;
+import io.questdb.cutlass.http.processors.RejectProcessorImpl;
+
+public class DefaultRejectProcessorFactory implements RejectProcessorFactory {
+    public static final RejectProcessorFactory INSTANCE = new DefaultRejectProcessorFactory();
+
+    @Override
+    public RejectProcessor getRejectProcessor(HttpConnectionContext context) {
```

Comment:
This can simply return HttpRequestProcessor

Reply:
can you explain this in more detail, Alex?
where should i put the methods from `RejectProcessor` interface then?

```
    void clear();
    boolean isRequestBeingRejected();
    HttpRequestProcessor rejectRequest(int code, CharSequence userMessage, CharSequence cookieName, CharSequence cookieValue, byte authenticationType);
```

Reply:
ah, you need `isRequestBeingRejected()`